components/cloak-visual.coffee

```diff
@@ -85,7 +85,7 @@ export default
 
 		# Use the width as the size if passed
 		sizes = if !props.sizes and props.width
-			if String(props.width).match /^\d+$/
+			if String(props.width)?.match /^\d+$/
```

Comment:
Huh ... we test for `props.width` right before ... and even with falsey width values, I don't get errors ... 

![image](https://user-images.githubusercontent.com/77567/166578826-c1882a65-a67e-419b-a98e-2c14177424bf.png)

I think would could probably ditch this change, yeah?  Or did was this definitely part of the solution?